docs/configuration_options.md

```diff
@@ -1,339 +1,339 @@
 # scylla-cluster-tests configuration options
```

Comment:
why there's so big change in this file?

Reply:
good question, I'll regenerate it again (it might still have big diff, since people forgot to generate when adding new options)

Reply:
now it's fixed, probably I did some manually editing, that I shouldn't have done